sample-shrinker.sh

```diff
@@ -456,6 +491,20 @@ while getopts 'b:B:c:x:paA:Sd:lo:nvh' opt; do
       fi
       minimum_bit_depth="${OPTARG}"
       ;;
+    r)
+      if ! [[ "$OPTARG" =~ ^[0-9]+$ ]]; then
+        .error "-r must be an integer; got invalid value: '$OPTARG'"
+        exit 1
+      fi
```

Comment:
Should we also support `44.1k` like sox does?  (We don't have to)

Reply:
Could - but out of scope for what I wanted to do this round. I do like the simplicity of integers